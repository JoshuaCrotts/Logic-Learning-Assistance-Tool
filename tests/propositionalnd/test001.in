// gay
// gay

(A -> B), A => B
