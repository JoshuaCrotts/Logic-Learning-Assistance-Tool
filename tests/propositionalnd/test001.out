1: (A -> B)                      P
2: A                             P
3: B                             Derived from [(A -> B), A], MP
∴ B  ■
